hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/util/FutureMessage.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.mesos.hdfs.util;
+
+import org.apache.mesos.Protos;
+import org.apache.mesos.SchedulerDriver;
+import org.apache.mesos.hdfs.scheduler.Scheduler;
+
+/**
```

Comment:
The state should be held in the parent class though. I think we can find a way to test and not add unnecessary code so I'm not convinced of the inner class.